examples/roam/src/components/Block.js

```diff
@@ -0,0 +1,383 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import ReactTextareaAutocomplete from '@webscopeio/react-textarea-autocomplete'
+import '@webscopeio/react-textarea-autocomplete/style.css'
+import { useEntity, useQuery, useTransact } from 'homebase-react'
+import debounce from 'lodash/debounce'
+import { nanoid } from 'nanoid'
+import React from 'react'
+import TextareaAutosize from 'react-autosize-textarea'
+import { Link } from 'react-router-dom'
+import RoamMarkdown from './RoamMarkdown'
+
+const Ref = ({ id }) => {
+  const [block] = useEntity(id)
+
+  return (
+    <div className="my-4">
+      <Link to={`/page/${block.get('_children', 0, 'uid')}`} className="text-blue-500">
```

Comment:
Wondering if we should add a comment about this syntax and what it's doing since I don't think we document it anywhere?